index.html

```diff
@@ -168,30 +167,42 @@
       "REC": "https://www.w3.org/TR/accname-aam-1.1/"
     },
 
-    preProcess: [ linkCrossReferences, mappingTables ],
+    preProcess: [ linkCrossReferences ],
 	postProcess: [ addPlatformMaintainers ],
 	xref: ["dom", "accname", "wai-aria", "infra"]
   }
 </script>
 <script type="text/javascript">
 
-var mappingTableLabels = {
-  viewByTable: "View as a single table",
-  expand: "Expand all",
-  collapse: "Collapse all",
-  showHideCols: "Show/Hide Columns:",
-  showActionText: "Show",
-  hideActionText: "Hide",
-  showToolTipText: "Show column",
-  hideToolTipText: "Hide column",
+document.addEventListener("DOMContentLoaded", () => {
+  var expandCollapseDetails = function (detCont, action) {
+    detCont.querySelectorAll('details').forEach(function (details) {
+      details.open = action !== 'collapse'
+    });
+  };
 
-  // map where keys are @id of associated mapping table:
-  viewByLabels: {
-    "role-mapping-table": "View by roles",
-    "state-property-mapping-table": "View by state/property",
-    "event-mapping-table": "View by state/property"
-  }
-}
+  document.querySelectorAll('button.expand').forEach(function (b){
+    b.addEventListener('click', function () {
+      detailsContainer = b.parentElement;
+      expandCollapseDetails(detailsContainer, 'expand');
+      b.disabled = true;
+      b.parentElement
+        .querySelector('button.collapse')
+        .removeAttribute('disabled');
+    });
+  });
+
+  document.querySelectorAll('button.collapse').forEach(function (b){
+    b.addEventListener('click', function () {
+      detailsContainer = b.parentElement;
+      expandCollapseDetails(detailsContainer, 'collapse');
+      b.disabled = true;
+      b.parentElement
+        .querySelector('button.expand')
+        .removeAttribute('disabled');
+    });
+  });
+});
```

Comment:
I just moved this code to here from the mapping tables file, I wonder if @pkra has an idea about where is should belong? here and duplicated across specs... or in the aria commons file still?

Also I'm fuzzy on the publication process, I have some vague idea we are suppose to not have javascript in the published draft.

Reply:
>  I wonder if @pkra has an idea about where is should belong? here and duplicated across specs... or in the aria commons file still?

Given how small it is now that, I personally don't mind either way - whichever the aam editors prefer.

> Also I'm fuzzy on the publication process, I have some vague idea we are suppose to not have javascript in the published draft.

I don't know but would observe that you're using an existing script tag; even if that script tag had "just" the configuration for the tables script, it seems to indicate that JS is ok.

Reply:
removed summary details, so don't need script after all